spec/lib/journaled/audit_log_spec.rb

```diff
@@ -0,0 +1,582 @@
+require 'rails_helper'
+
+RSpec.describe Journaled::AuditLog do
+  let(:auditable_model) do
+    Class.new do
+      extend ActiveModel::Callbacks
+      define_model_callbacks :create, :update, :destroy, only: %i(after)
+
+      include Journaled::AuditLog
+
+      def self.table_name
+        'objects'
+      end
+
+      def id
+        'random-id'
+      end
+
+      def created_at
+        'very-recently'
+      end
+
+      def updated_at
+        'close-to-now'
+      end
+
+      def attributes
+        { all: 'attributes', password: true }
+      end
+
+      if Rails::VERSION::MAJOR >= 7
+        def encrypted_attribute?(_key)
+          false
+        end
+      end
+    end
+  end
+
+  before do
+    stub_const('MyModel', auditable_model)
+  end
+
+  around do |example|
+    Journaled.tagged(request_id: 123) do
+      example.run
+    end
+  end
+
+  describe '.default_ignored_columns' do
+    it 'defaults to timestamps, but is configurable' do
+      expect(described_class.default_ignored_columns).to eq %i(created_at updated_at)
+      described_class.default_ignored_columns = []
+      expect(described_class.default_ignored_columns).to eq []
+    ensure
+      described_class.default_ignored_columns = %i(created_at updated_at)
+    end
+  end
+
+  describe '.default_stream_name' do
+    it 'defaults to primary default, but is configurable' do
+      # TODO: When upstreaming, check that it is set to Journaled.default_stream_name by default
```

Comment:
Oops! Pushed a fix -- I think all I meant was that I wanted to follow the other pattern of `expect`ing the value to be something, rather than holding onto a `_was` value.